guides/V1_ActionsEvents.md

```diff
@@ -0,0 +1,183 @@
+
+![AppConfig Usage Screenshot](../images/AddAction2.png)
+
+5. click Add configuration handler.
+
+![AppConfig Usage Screenshot](../images/AddAction3.png)
+
+6. See a confirmation of the addition.
+
+![AppConfig Usage Screenshot](../images/AddAction4.png)
+
+7. You will see your Client Action Handler...
+
+![AppConfig Usage Screenshot](../images/AddAction5.png)
+
+8. ...which you can Edit or Delete in the future.
+
+![AppConfig Usage Screenshot](../images/AddAction6.png)
+
+## Things to consider when you have an explicit action handler
+
+It's important to think about these things when adding an explicit action handler to your App.
+
+ - The explicit action can be requested by anyone, so the caller and the provided parameters must be verified.
+
+ - If your explicit action requires knowledge of the user, then you must obtain the user id and name also to establish a link between the user's ID and the user's name.
+
+ - If your explicit action takes into account something to do with the space, then you should display the space name to help the caller verify that it is indeed the space for which this action should take place.
+
+ - You should display a continue to space button to complete the process.
+
+ Let's walk through these.
+
+### Protect the explicit action endpoint
+
+Making an endpoint publicly available has security risks; for example, your endpoint could receive requests from malicious sources. To minimize risk:
+- Be prepared for <a href="https://en.wikipedia.org/wiki/Network_security#Types_of_Attacks" target="_blank">network attacks</a> and ensure that appropriate counter-measures are in place.
+- Do not assume that the request or its parameters are trustworthy until you have verified the call.
+
+
+### Verify the call
+
+Use the url parameter `actionToken` and utilize Watson Work Services to verify the call.
+Authorize your app [as an App](../references/V1_oauth_token_client_credentials.yml) and access the action data with the `actionToken` - see [Get Action Context](./V1_GetActionContext.md).
+If the access to the action data is successful, then you can be sure that the call has been triggered by Watson Workspace.
+You will also get the action context data, which may include things like `spaceId` and `userId`.  `spaceId` is the unique ID of the space for which the action has been triggered.  While `userId` is the unique ID of the user who triggers the action.
+
+<div class="tip">
+  <img src="https://github.com/watsonwork/watsonwork-developer-docs/images/note_pencil.png" />
+  <p><strong>Note:</strong> If the implementation of the your explicit action handler page is still based on a version prior to 1.1.0 of the <a href="https://developer.watsonwork.ibm.com/docs/apps/get-action-context>Get Actiopn Context API</a>, then you have to do the oauth flow to verify the caller as a user of Watson Work Services, to obtain the user id and name, and to establish a link between the user's ID and the user's name.</p>
```

Comment:
Not sure what you meant here

Reply:
this is referring to prior way of doing things and that had a inflection point around 1.1.0